Detectors/HMPID/workflow/src/PedestalsCalculationSpec.cxx

```diff
@@ -41,6 +41,9 @@
 #include "Framework/Task.h"
 #include "Framework/WorkflowSpec.h"
 #include "Framework/Logger.h"
+#include "Framework/RawDeviceService.h"
+#include "Framework/CommonServices.h"
+#include <fairmq/Device.h>
```

Comment:
Do we need all the includes? Isn't RawDeviceService enough?

Reply:
I removed the CommonService.h include